Source/Urho3D/Input/Input.cpp

```diff
@@ -167,7 +174,13 @@ void EmscriptenInput::RequestPointerLock(MouseMode mode, bool suppressEvent)
 {
     requestedMouseMode_ = mode;
     suppressMouseModeEvent_ = suppressEvent;
+#ifdef URHO3D_CUSTOM_SHELL

```

Comment:
When do we have "custom" shell and when is it "standard"?
Is "standard" shell even supported?

Reply:
I'll be honest. I'm trying to keep the support for the legacy emscripten shell as it was before until I resolve all the important issues with the new shell. Once that's done, I'm planning to remove it completely.